### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by emmiiorji**
This change configures Redux Store and writes actions and reducer for adding and removing books.

Actions defined include:

- those for adding and removing a book whose initial state is set as an empty array of books.
- one for checking the status of categories (returning a string "Under Construction") whose initial state is set to be an empty array of categories.

In all cases, the current state is returned when the action is unknown.

### CODE REVIEW

File: src/redux/configureStore.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import books from './books/books';
+import categories from './categories/categories';
+
+const rootReducer = combineReducers({
```
Comment (mateo951) [inline src/redux/configureStore.js:6]: Uses `combineReducers` to create a reducer ✔

File: src/redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import books from './books/books';
+import categories from './categories/categories';
```
Comment (mateo951) [inline src/redux/configureStore.js:4]: Great! Well done following the ducks pattern when importing your files 💪👀

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD = 'bookstore/book/ADD';
+const REMOVE = 'bookstore/book/remove';
```
Comment (mateo951) [inline src/redux/books/books.js]: Defines `ADD_BOOK` and `REMOVE_BOOK` as action types ✔

Comment (mateo951) [reply src/redux/books/books.js]: Please be sure to follow the naming conventions on your `REMOVE` action as well 🤓 If you want a bit more information in the pattern and conventions check the following [link](https://decembersoft.com/posts/a-simple-naming-convention-for-action-creators-in-redux-js/) 📝

File: src/redux/books/books.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      return state.filter((book) => book.id !== payLoad.id);
+    default:
+      return state;
+  }
+};
+
+// Action Creators
+export const addBook = (book) => ({ type: REMOVE, payLoad: { book } });
+
+export const removeBook = (id) => ({ type: ADD, payLoad: { id } });
```
Comment (mateo951) [inline src/redux/books/books.js]: You create and export the **Action Creators** 😎 Great job!

File: src/redux/books/books.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+export default reducer;
```
Comment (mateo951) [inline src/redux/books/books.js:25]: You created a reducer and export it default 📝🧐 Awesome ✔

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @emmiiorji, 

#### 🚀 Highlights 🚀
- The structure follows the `ducks pattern` ✔ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/redux/books/books.js [hunk 6]
```diff
@@ -1,23 +1,25 @@
 // Actions
-const ADD = 'bookstore/book/ADD';
-const REMOVE = 'bookstore/book/remove';
+const Actions = {
+  ADD_BOOK: 'bookstore/book/ADD',
+  REMOVE_BOOK: 'bookstore/book/remove',
```
Comment (mateo951) [inline src/redux/books/books.js]: Excellent ✔ The constant follows the naming convention! I think I didn't explain myself well 😥 I will approve your changes even though we need to change one more thing 🤓 

Your type action:
       
    REMOVE_BOOK: 'bookstore/book/remove',

Intended:

    REMOVE_BOOK: 'bookstore/book/REMOVE',

**Review by mateo951 — APPROVED** [review]
### Hi @emmiiorji, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 📝 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 6 above)
Comment (emmiiorji) [reply src/redux/books/books.js]: Hi @mateo951 
Thanks for a kind review. I'll modify the naming for the REMOVE action as requested.

**Pull request merged**